lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -439,7 +439,7 @@ void ProxyRenderDelegate::_InitRenderDelegate(MSubSceneContainer& container) {
                 globalSelection->addObserver(_observer);
             }
 
-            Ufe::Scene::instance().addObjectAddObserver(_observer);
+            Ufe::Scene::instance().addObserver(_observer);
```

Comment:
Do we need to do something in the observer now that we'll receive notifications other than object add?  I.e. dynamic cast the notification and early out if not object add?

Reply:
Answered my own question: no, because the observer is doing a dynamic_cast for the appropriate notification types already, at line 234.